libs/service-portal/access-control/delegations/src/screens/AccessControl.tsx

```diff
@@ -0,0 +1,104 @@
+import { useHistory } from 'react-router-dom'
+import { useLocation } from 'react-use'
+import * as kennitala from 'kennitala'
+import { Box, Tabs } from '@island.is/island-ui/core'
+import {
+  IntroHeader,
+  m,
+  ServicePortalPath,
+} from '@island.is/service-portal/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { useAuth } from '@island.is/auth/react'
+import { isDefined } from '@island.is/shared/utils'
+import { DelegationsIncoming } from '../components/delegations/incoming/DelegationsIncoming'
+import { DelegationsOutgoing } from '../components/delegations/outgoing/DelegationsOutgoing'
+import { Features, useFeatureFlag } from '@island.is/react/feature-flags'
+
+const TAB_DELEGATION_OUTGOING_ID = '0'
+const TAB_DELEGATION_INCOMING_ID = '1'
```

Comment:
~These could be more descriptive, right?

```suggestion
const TAB_DELEGATION_OUTGOING_ID = 'outgoing'
const TAB_DELEGATION_INCOMING_ID = 'incoming'
```